### ISSUE TITLE
[A11y] Desktop/Mobile Search Results List: Image Thumbnail Labels

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1833]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Search

### Admin user?

No

### Issue Description

Search Results List: https://staging.oregondigital.org/catalog?utf8=%E2%9C%93&search_field=all_fields&q%5B%5D=test

3 images in the results labels have the following:

Failed Instance:
label: Ensures every form element has a label

Image 1
Path: #batch_document_9z902z939
Snippet:
`<input type="checkbox" name="batch_document_ids[]" id="batch_document_9z902z939" value="9z902z939" class="batch_document_selector">`

Image 2
Path: #batch_document_kw52j8067
Snippet:
`<input type="checkbox" name="batch_document_ids[]" id="batch_document_kw52j8067" value="kw52j8067" class="batch_document_selector">`

Image 3
Path: #batch_document_1544bp11q
Snippet:
`<input type="checkbox" name="batch_document_ids[]" id="batch_document_1544bp11q" value="1544bp11q" class="batch_document_selector">`

### Suggested Fix

Fix ONE of the following:

- Form element does not have an implicit (wrapped) <label>
- Form element does not have an explicit <label>
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute
- Element has no placeholder attribute
- Element's default semantics were not overridden with role="none" or role="presentation"

**Comment by CGillen** [comment]
Fixed by #2059
